add markdown evaluation to labels

irauschek opened the pull request:
Added missing markdown evaluation of checkbox labels within data.html.twig if activated. To be able to send the agreed terms of condition as link within an email - see #227 and #253
Translate the label into the current language set in the admin interface user preferences if defined.
Furthermore added the possibility to use markdown syntax within labels of checkboxes, date, display, email, file, month, password, radio, range, select_optgroup, select, tel, text, textarea, and time too - see #281

rhukster commented:
I feel like we need to simplify things a bit in forms.. slight changes should not have this big an impact.  I'm going to be reviewing forms twig a bit next week.

mahagr commented:
I've started to use `{% use ... %}` to override common blocks in a single file, which seems to be a good match for this case.

irauschek commented:
Despite the missing markdown evaluation of checkbox labels within data.html.twig if activated, to be able to send the agreed terms of condition as link within an email - see #227 and #253, all other addressed issues have been solved in release 4.0.0
